app/src/features/onboarding/components/auth/components/Form/components/SSOSignInForm/index.tsx

```diff
@@ -0,0 +1,111 @@
+import React, { useCallback, useState } from "react";
+import { BiArrowBack } from "@react-icons/all-files/bi/BiArrowBack";
+import { AuthFormInput } from "../AuthFormInput";
+import { RQButton } from "lib/design-system/components";
+import APP_CONSTANTS from "config/constants";
+import { getFunctions, httpsCallable } from "firebase/functions";
+import Logger from "lib/logger";
+import { MdOutlineWarningAmber } from "@react-icons/all-files/md/MdOutlineWarningAmber";
+import { getDomainFromEmail, getEmailType, isEmailValid } from "utils/FormattingHelper";
+import { toast } from "utils/Toast";
+import { trackLoginAttemptedEvent } from "modules/analytics/events/common/auth/login";
+import { AUTH_PROVIDERS } from "modules/analytics/constants";
+import "./index.scss";
+
+interface Props {
+  email: string;
+  setAuthMode: (authMode: string) => void;
+  setEmail: (userEmail: string) => void;
+  source: string;
+}
+
+export const SSOSignInForm: React.FC<Props> = ({ setAuthMode, email, setEmail, source }) => {
+  const [isLoading, setIsLoading] = useState(false);
+  const [isNoConnectionFoundCardVisible, setIsNoConnectionFoundCardVisible] = useState(false);
+  const domain = getDomainFromEmail(email);
+
+  const handleCaptureSSOInterest = useCallback(() => {
+    if (!email) {
+      toast.error("Please enter your email");
+      return;
+    }
+
+    if (!isEmailValid(email)) {
+      toast.error("Please enter a valid email");
+      return;
+    }
+    trackLoginAttemptedEvent({
+      auth_provider: AUTH_PROVIDERS.SSO,
+      email,
+      place: window.location.href,
+      email_type: getEmailType(email),
+      domain: getDomainFromEmail(email),
+      source,
+    });
+
+    setIsLoading(true);
+    const captureSSOInterest = httpsCallable(getFunctions(), "auth-captureSSOInterest");
+
+    captureSSOInterest({ email })
```

Comment:
What if `success:false` is returned?

Reply:
The end result in UI is going to be the same for all cases for now even if interest capture is failed.